Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/ScheduleDatabase.java

```diff
@@ -25,29 +25,6 @@
     private static SQLiteDatabase sDB;
```

Comment:
Can this file just be deleted?  Or will we ever need a sqldb locally?

Reply:
FAQ still being retrieved from SQLite, but renamed the file to reflect this and moved the schedule items.